### PR TITLE
chore: Add donation information info

### PR DESCRIPTION
**Pull request by Wandarry**
Add donation information

https://github.com/Wandarry/VitaNova/assets/101728187/e3d68790-7dc9-4f83-bf75-b7ab36c65e05

### CODE REVIEW

File: utils/validationSchemas.ts [hunk 1]
```diff
@@ -47,3 +47,12 @@ export const registerValidationSchemaThirdForm = object({
     "Entrez le même mot de passe que le premier champ",
   ),
 });
+
```
Comment (BOCOVO) [inline utils/validationSchemas.ts]: is these lines related to the donation form ?

Comment (Wandarry) [reply utils/validationSchemas.ts]: No...
I need a new validation schema for each categorie, right ?

Comment (BOCOVO) [reply utils/validationSchemas.ts]: Yes I think
